src/monorepo-workflow-operations.ts

```diff
@@ -137,118 +107,15 @@ export async function followMonorepoWorkflow({
     project,
     releaseSpecificationPath,
   );
-  const releasePlan = await planRelease(
+  const releasePlan = await planRelease({
     project,
     releaseSpecification,
-    releaseSpecificationPath,
-  );
-  await applyUpdatesToMonorepo(project, releasePlan, stderr);
-  await removeFile(releaseSpecificationPath);
-  await captureChangesInReleaseBranch(project, releasePlan);
-}
-
-/**
- * Uses the release specification to calculate the final versions of all of the
- * packages that we want to update, as well as a new release name.
- *
- * @param project - Information about the whole project (e.g., names of packages
- * and where they can found).
- * @param releaseSpecification - A parsed version of the release spec entered by
- * the user.
- * @param releaseSpecificationPath - The path to the release specification file.
- * @returns A promise for information about the new release.
- */
-async function planRelease(
-  project: Project,
-  releaseSpecification: ReleaseSpecification,
-  releaseSpecificationPath: string,
-): Promise<ReleasePlan> {
-  const today = getToday();
-  const newReleaseName = today.toISOString().replace(/T.+$/u, '');
-  const newRootVersion = [
-    today.getUTCFullYear(),
-    today.getUTCMonth() + 1,
-    today.getUTCDate(),
-  ].join('.');
-
-  const rootReleasePlan: PackageReleasePlan = {
-    package: project.rootPackage,
-    newVersion: newRootVersion,
-    shouldUpdateChangelog: false,
-  };
-
-  const workspaceReleasePlans: PackageReleasePlan[] = Object.keys(
-    releaseSpecification.packages,
-  ).map((packageName) => {
-    const pkg = project.workspacePackages[packageName];
-    const versionSpecifier = releaseSpecification.packages[packageName];
-    const currentVersion = pkg.validatedManifest.version;
-    let newVersion: SemVer;
-
-    if (versionSpecifier instanceof SemVer) {
-      const comparison = versionSpecifier.compare(currentVersion);
-
-      if (comparison === 0) {
```

Comment:
I've moved these checks to when the release spec is validated, as it makes more sense there.